README.md

```diff
@@ -44,3 +44,25 @@ For interactive login, just run `banzai login`, and follow the instructions give
 ### Use
 
 See [command reference](https://banzaicloud.com/docs/pipeline/cli/reference/) in the [official documentation](https://banzaicloud.com/docs/pipeline/cli/).
+
+### Use Docker image
+
+Logging in
```

Comment:
We might add a version where no config file is mounted, and the base url and the token come from an env var.

Reply:
https://github.com/banzaicloud/banzai-cli/issues/305